### ISSUE TITLE
UIKitCore con < iOS 11

### ISSUE THREAD

**Issue by diegogarciar** [alexisakers/BulletinBoard#180]
**Problem Description:**
Hello, a user has iOS 11 and app is crashing on

```
BulletinCloseButton ->configureSubviews
accessibilityLabel = Bundle.UIKitCore.localizedString(forKey: "Close", value: "Close", table: nil)

```
with an error of 

> Fatal error: Unexpectedly found nil while unwrapping an Optional value.

 It seems you're force unwrapping `Bundle(identifier: "com.apple.UIKitCore")!` at Bundle.UIKitCore but it's not available on older OS? Anyway, a simple optional might be enough.

**Steps to reproduce:**
present any bulletin using an older iOS version

**Environment:**
 - Device: iPhone 5 and 8
 - OS: iOS 10.3.4 and iOS 11.4.1
 - Version of BulletinBoard: 4.1.1

**Comment by diegogarciar** [comment]
I see there's already a pull request for this fix, We can close this issue when it goes to master

**Comment by alexaubry** [comment]
The fixed shipped in 5.0, thanks for your patience.
